modules/gapi/src/backends/onnx/bindings_onnx.cpp

```diff
@@ -8,6 +8,19 @@ cv::gapi::onnx::PyParams::PyParams(const std::string& tag,
                                    const std::string& model_path)
     : m_priv(std::make_shared<Params<cv::gapi::Generic>>(tag, model_path)) {}
 
+cv::gapi::onnx::PyParams& cv::gapi::onnx::PyParams::cfgMeanStd(const std::string &layer_name,
```

Comment:
Are we able to pass per-channel parameters here?

From PyTorch:

```
        self.transform = transforms.Compose([
            transforms.ToTensor(),
            transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
        ])
```

Reply:
@asutic yes as in the above example:
```python
net.cfgMeanStd('data_0', [0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
```